code/game/turfs/space/space.dm

```diff
@@ -16,7 +16,7 @@
 
 
 /turf/open/space/New()
-	update_icon()
+	icon_state = SPACE_ICON_STATE
```

Comment:
Is there any reason we need to do this in `New()` can't we do this elsewhere? Something Something proc being called for each space turf.